### PR TITLE
Scala 2.13.14 draft release notes

### PR DESCRIPTION
**Pull request by SethTisue**
rendered: https://github.com/SethTisue/scala-dev/blob/scala-2.13.14/releases/2.13.14.md

feedback welcome

### CODE REVIEW

**Review by Philippus — APPROVED** [review]

File: releases/2.13.14.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Scala 2.13.14 (DRAFT)
+
+The Scala team at Lightbend is pleased to announce Scala 2.13.14.
+
+The following changes are highlights of this release:
+
+### Align with Scala 3
+
+* Introduce `-Xsource-features`, for customizing the behavior of `-Xsource:3` and `-Xsource:3-cross` ([#10709](https://github.com/scala/scala/pull/10709) by [@som-snytt](https://github.com/som-snytt))
```
Comment (som-snytt) [inline releases/2.13.14.md]: This should say lrytz. (The comment on Mar 12 explains.) If anything, I just slowed him down.

**Review by som-snytt — CHANGES_REQUESTED** [review]

**Review by som-snytt — APPROVED** [review]

File: releases/2.13.14.md (hunk 1 above)
Comment (lrytz) [reply releases/2.13.14.md]: We like people that slow others down in Switzerland (no irony!), that's the core of our political system :)

File: releases/2.13.14.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Scala 2.13.14 (DRAFT)
+
+The Scala team at Lightbend is pleased to announce Scala 2.13.14.
+
+The following changes are highlights of this release:
+
+### Align with Scala 3
+
+* Introduce `-Xsource-features`, for customizing the behavior of `-Xsource:3` and `-Xsource:3-cross` ([#10709](https://github.com/scala/scala/pull/10709))
```
Comment (lrytz) [inline releases/2.13.14.md:9]: Add a link to https://docs.scala-lang.org/scala3/guides/migration/tooling-scala2-xsource3.html (with the updates in https://github.com/scala/docs.scala-lang/pull/2994)

**Comment by sjrd** [comment]
It might be worth mentioning that the 5 fake source files of `library-aux` are now part of the sources jar. It can impact tools that use the source jar in a meaningful way. For example the Scala.js build needs to explicitly exclude those files using
```scala
scalalibInternal.v2_13/Compile/sources ~= {
  val excludes = Set("Any.scala", "AnyRef.scala", "Nothing.scala", "Null.scala", "Singleton.scala")
  prev => prev.filterNot(f => f.getParentFile.getName == "scala" && excludes.contains(f.getName))
}
```
I wonder if IDEs will actually be able to deal with those or if they will panic. Have we checked with IntelliJ and Metals?

**Comment by lrytz** [comment]
IntelliJ was tested (https://github.com/scala/scala/pull/10714), including the sources was actually a request that came through IntelliJ (https://youtrack.jetbrains.com/issue/SCL-22167).

For metals I tried a small sbt project with
```
resolvers += "scala-integration" at "https://scala-ci.typesafe.com/artifactory/scala-integration/"
ThisBuild / scalaVersion := "2.13.14-bin-a527019"
```

While I can navigate to `Unit` for example
image


I cannot navigate to the new sources like AnyRef / Null.

@tgodzik is that worth a ticket?

**Comment by tgodzik** [comment]
We might need to first release metals for that versions, which requires semanticdb. We can confirm once everything is out.

**Comment by SethTisue** [comment]
If anyone is trying out `-Xsource-features`, be sure and look at the doc revisions at https://github.com/scala/docs.scala-lang/pull/2994 — the PR will be merged at announcement time.

**Comment by SethTisue** [comment]
live now at https://github.com/scala/scala/releases/tag/v2.13.14

**Pull request closed without merging**